src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/service/AddressLookupService.kt

```diff
@@ -5,6 +5,15 @@ import uk.gov.justice.digital.hmpps.historicalprisonerapi.repository.AddressLook
 
 @Service
 class AddressLookupService(private val addressLookupRepository: AddressLookupRepository) {
-  fun findPrisonersWithAddresses(addressTerms: String): List<String> =
-    addressLookupRepository.findPrisonersWithAddresses("NEAR(($addressTerms), 5, TRUE)")
+  fun findPrisonersWithAddresses(addressTerms: String): List<String> {
+    // duplicate logic from HPA front end
+    val searchTerm = addressTerms
```

Comment:
original logic can be found at https://github.com/ministryofjustice/hmpps-hpa/blob/119ad24d352cd3fcb258b24caae96e25eb63f9fb/data/utils.js#L116-L121